tests/sentry/auth/test_superuser.py

```diff
@@ -170,6 +170,21 @@ def test_su_access_logs(self, logger):
             },
         )
 
+    def test_max_time_org_change_within_time(self):
+        request = self.build_request()
+        request.organization = self.create_organization(name="not_our_org")
+        superuser = Superuser(request, allowed_ips=())
+
+        assert superuser.is_active is True
+
+    def test_max_time_org_change_time_expired(self):
```

Comment:
doing something like `freeze_time` instead of setting the expiry on the su object itself is more realistic to the flow of a user, so converting the test to do that is preferable.